src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
+  margin: auto;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  box-sizing: border-box;
```

Comment:
Avoid using absolute positioning to center elements. This can lead to issues with responsiveness. Consider using flexbox or grid on the parent element to achieve centering.